tequilapi/client/client.go

```diff
@@ -514,3 +516,35 @@ func (client *Client) Beneficiary(address string) (res contract.IdentityBenefici
 	err = parseResponseJSON(response, &res)
 	return res, err
 }
+
+// SetMMNApiKey sets MMN's API key in config and registers node to MMN
+func (client *Client) SetMMNApiKey(data contract.MMNApiKeyRequest) error {
+	response, err := client.http.Post("mmn/api-key", data)
```

Comment:
you're not checking the error here.

Reply:
@vkuznecovas this is because if the request fails with a non-200 (e.g. 422) status code, the error generated is super generic. I need to parse the response of a 422 or 403 status code request to get the validation error messages from MMN. I understand the confusion, but I don't see another way of doing it. Advice is welcome.